src/test/java/com/puppycrawl/tools/checkstyle/AuditEventDefaultFormatterTest.java

```diff
@@ -104,6 +104,23 @@ public void testFormatModuleNameDoesNotContainCheckSuffix() {
     }
 
     @Test
+    public void testFormatModuleWithModuleId() {
+        final AuditEvent mock = PowerMockito.mock(AuditEvent.class);
```

Comment:
It was my mistake to allow such code to appear, mocks should not be used in simple cases like this.
new issue - #4931